asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -199,5 +210,161 @@ NOTE: This requirement supplements RR1162 in <<ref_ieee_11073_10700_2022>>: _The
+
+There is no overlap in timestamps when a non-slewing adjustment shifts the device clock forward in time. 
+
+image::vol1-diagram-use-case-stad-ns-forward.svg[align=center]
+
+When a non-slewing adjustment shifts the device's time-frame reference back in time, only timestamps before the last timestamp recorded in the MDIB from epoch 0 belong unambiguously to the new time-frame reference.
+
+image::vol1-diagram-use-case-stad-ns-back.svg[align=center]
+
+When a device experiences multiple non-slewing adjustments in a short period of time, the earliest timestamp unambiguously in the current time-frame reference may be from an earlier epoch. 
+
+image::vol1-diagram-use-case-stad-ns-back-forth.svg[align=center]
+
+.R1561
+[sdpi_requirement#r1561,sdpi_req_level=may]
+****
+The <<vol1_spec_sdpi_p_actor_somds_provider>> may indicate a timestamp belongs to a specific epoch using the SDPi epoch extension. 
+
+.Notes
+[NOTE]
+[%collapsible]
+====
+Timestamps for states that are not updated frequently may be versioned with a specific epoch. 
+
+====
+****
+
+.R1562
+[sdpi_requirement#r1562,sdpi_req_level=shall]
+****
+The <<vol1_spec_sdpi_p_actor_somds_provider>> shall report epoch adjustments from the earliest versioned timestamp's time-frame referenced in the MDIB to the current time-frame reference.  
+
+.Notes
+[NOTE]
+[%collapsible]
+====
+Epoch adjustments are optionally provided for <<vol1_spec_sdpi_p_actor_somds_participant>>s to adjust historic data. All relevant adjustments must be provided when versioned timestamps are included in an MDIB. 
 
+====
+****
+
+// This is for the sledge hammer approach. I figure out what a universal rule could be or how to communicate epoch changes
+// across MdibVersionGroup/@SequenceId since it seems that any information inside the MDS implicitly is scoped to the 
+// sequence id. 
+.R1566
+[sdpi_requirement#r1566,sdpi_req_level=shall]
+****
+The <<vol1_spec_sdpi_p_actor_somds_provider>> that changes the MDIB sequence identifier when it can no longer make smooth adjustments to its time-frame reference shall consider the risks arising from gaps in continuous data. 
```

Comment:
Shouldn't the requirement subject be the MANUFACTURER of the Provider?

Reply:
Yes; thanks.